.changeset/seven-jars-yell.md

```diff
@@ -0,0 +1,9 @@
+---
+'@graphql-inspector/core': major
```

Comment:
We should also add instructions on how to achieve the previous output, without showing nested change output? Maybe also adjust the docs accordingly? Also people using the CLI, should be able to only output non-nested changes 🤔